src/Traits/Resource/ResourceModelActions.php

```diff
@@ -33,8 +33,13 @@ public function export(): ?ExportHandler
             return null;
         }
 
-        return ExportHandler::make(__('moonshine::ui.export'))
-            ->csv();
+        $handler =  ExportHandler::make(__('moonshine::ui.export'));
+
+        if (config('moonshine.export.format') !== 'xlsx') {
```

Comment:
мне кажется так лучше
```suggestion
        if (config('moonshine.export.format') === 'csv') {
```